var/spack/repos/builtin/packages/gmp/package.py

```diff
@@ -41,7 +42,7 @@ class Gmp(AutotoolsPackage):
 
     def configure_args(self):
         args = ['--enable-cxx']
-        # We need this flag if we want all the following checks to pass.
+        # This flag is necessary for the Intel build to pass `make check`
```

Comment:
The checks no longer follow now that we switched to `AutotoolsPackage`, so I updated this comment to be more self-explanatory.